src/bin/wasm-tools/validate.rs

```diff
@@ -211,28 +184,27 @@ fn parse_features(arg: &str) -> Result<WasmFeatures> {
         };
         match part {
             "all" => {
-                for (name, feature) in FEATURES {
-                    // don't count this under "all" for now.
-                    if *name == "deterministic" {
-                        continue;
-                    }
```

Comment:
You don't need to keep this deterministic exclusion?

Reply:
Ah yeah this was an old feature of wasmparser that was removed awhile back and this was forgotten to be removed at the time.